ethcore/src/state/mod.rs

```diff
@@ -486,7 +486,7 @@ impl State {
 		let mut addresses = self.cache.borrow_mut();
```

Comment:
`commit_cache` -> what does this even mean? a "cache" cannot be committed - it is simply a speedily accessible version of what is on a secondary storage device (i.e. disk). if it's something that has changed without its backing store then it's a "buffer".